docs: briefly mention activities calling activities (#840)
temporalio/documentation: docs/typescript/activities.md

You can [read the tests](https://github.com/temporalio/sdk-typescript/blob/7d47f501cb56cced27118b5f0abb320cc0ba03ef/packages/test/src/test-async-completion.ts#L40-L98) for more information.

> ## Activities calling Activities

Reviewer: ```suggestion
### Activities calling Activities
```

It may sometimes seem convenient in code to have an activity call another activity.
We generally recommend not to: activities are regular functions, so calling to one directly will not be seen -and therefore logged- by the Temporal service.

> Instead, prefer moving logic out of the activities so that the parent workflow uses the result of one activity to call the other activity.

Reviewer 2: Rephrase suggestion: "Activities are regular functions. Calling another Activity from within an Activity will not be logged by the Temporal service, and is therefore not a recommended practice. Instead, move logic out of the Activities so that the parent Workflow uses the result of one Activity to call the other Activity."

> Instead, prefer moving logic out of the activities so that the parent workflow uses the result of one activity to call the other activity.

Reviewer 3: ```suggestion
Instead, move logic out of the Activities and have the parent Workflow use the result of one Activity to call the other Activity.
```
